README.md

```diff
@@ -357,6 +359,8 @@ webidl2js is implementing an ever-growing subset of the Web IDL specification. S
 - `[TreatNullAs]`
 - `[Unforgeable]`
 - `[Unscopable]`
+- `[CEReactions]`
+- `[HTMLConstructor]`
```

Comment:
I'd open a new section for these two as "Web IDL extensions defined in HTML."

Reply:
Perhaps also say that these are "supported" in the sense that the integrator can use hooks to properly modify their behavior?